content/_data/opencollective.js

```diff
@@ -2,48 +2,106 @@
 import eleventyFetch from '@11ty/eleventy-fetch';
 import { groupBy } from 'lodash-es';
 
+const GITHUBKEY = '';
+
 const FilteredProfiles = [
   // if there are backers we need to exclude…
 ];
 
-const TIERS = ['Great Horned Owl', 'Blue-Footed Booby', 'Common Loon'];
+const TIERS = [
+  'Great Horned Owl',
+  'Blue-Footed Booby',
+  'Common Loon',
+  'GitHub',
+];
 
 const getDefaultAvatarUrl = (url) => {
   const slug = url.split('/').at(-1);
   return slug ? `https://images.opencollective.com/${slug}/avatar.png` : null;
 };
 
+const loadGithubSponsors = async () => {
+  const url = 'https://api.github.com/graphql';
+  const { data } = await eleventyFetch(url, {
+    type: 'json',
+    duration: '0s',
+    fetchOptions: {
+      method: 'POST',
+      headers: {
+        Authorization: `Bearer ${GITHUBKEY}`,
+      },
+      body: JSON.stringify({
+        query: `
+        {
+  organization(login: "oddbird") {
+    name
+    sponsors(first: 100) {
+      totalCount pageInfo { hasNextPage }
+      nodes {
+        __typename
+        ... on User { name avatarUrl websiteUrl url }
+        ... on Organization { name avatarUrl websiteUrl url }
```

Comment:
I don't have access to the OddBird org to test this query, but from the [Explorer](https://docs.github.com/en/graphql/overview/explorer) it looks like this should be valid and get us the info we need:

```graphql
{
  organization(login: "oddbird") {
    sponsors {
      nodes {
        ... on User {
          sponsorshipForViewerAsSponsor {
            isOneTimePayment
            isActive
            tier {
              id
              description
            }
          }
        }
        ... on Organization {
          sponsorshipForViewerAsSponsor {
            isOneTimePayment
            isActive
            createdAt
            tier {
              id
              description
            }
          }
        }
      }
    }
  }
}
```

Reply:
@jerivas `sponsorshipForViewerAsSponsor` won't work- the `viewer` is the token's owner, so if we use OddBird's token, it would be "show me sponsorships where OddBird is sponsoring OddBird".

Reply:
Here's my full running query. I'm still getting no results, and am unsure in `sponsorshipForViewerAsSponsorable` if the `viewer` is "jamesnw" (since I own the token I'm using for testing) or "oddbird". If it's the token owner, I think we'd need to set up the App, since I don't think you can have a Personal Access Token for an organization.

```
{
  organization(login: "oddbird") {
    sponsors(first: 100) {
      nodes {
        ... on User {
          sponsorshipForViewerAsSponsorable(activeOnly: false) {
            isOneTimePayment
            isActive
            tier {
              id
              description
            }
          }
        }
        ... on Organization {
          sponsorshipForViewerAsSponsorable(activeOnly:false) {
            isOneTimePayment
            isActive
            createdAt
            tier {
              id
              description
            }
          }
        }
      }
    }
  }
}
```

Reply:
I'm fine going the GH App route if that's the right way. [This answer](https://github.com/orgs/community/discussions/24649#discussioncomment-7766153) mentions using sponsorshipsAsMaintainer: https://github.com/xmonad/xmonad-web/blob/0407056255cc253da74a586ef80c4ed16c51d75e/_sponsors.sh

Reply:
It ended up being `sponsorshipsAsMaintainer` with `isActive:false`. The one Github sponsor was a one-time sponsor. They donated on May 22, and so they were active for exactly a month, and dropped off the list on Saturday.